src/stores/dataStore.js

```diff
@@ -86,8 +86,7 @@ export class DataStore {
         location_ids: getLocationIdsByNode(
           toJS(configStoreInstance.selectedLocationNodes)
         ),
-        selected_metadata_fields:
-          configStoreInstance.getSelectedMetadataFields(),
+        selected_metadata_fields: configStoreInstance.getSelectedMetadataFields(),
```

Comment:
What are your JS formatter settings? Let me know because we should settle on one to avoid constant changes back and forth.

I'm using https://prettier.io/ with settings:

```
"prettier.singleQuote": true,
"prettier.semi": false,
"editor.formatOnSave": true,
"prettier.eslintIntegration": true,
```

Reply:
I've been using the prettier plugin for atom but hadn't configured it. I added a local config file with your settings to my machine so this should be resolved.